### ISSUE TITLE
Test RSC SureChembl linksets

### ISSUE THREAD

**Issue by stain** [openphacts/IdentityMappingService#32]
@nicklynch  said

> RSC have created a linkset for a 100 K mini dataset for SureChMBL
> Linkset download for 100K http://ops.rsc.org/download/RDF-2016.01.12.zip 

> There is properties in this but we should not load ttl file that for now as not sure of licensing of the ACD/Labs for this
> Would it be possible to load this into Dev IMS so we can start some testing on this and the API calls?

I've made https://github.com/openphacts/ops-rsc-surechembl-dataset that captures this and produces the linksets only at http://repository.mygrid.org.uk/artifactory/ops/org/openphacts/data/ops-rsc-linksets/0.20151104.0-SNAPSHOT/

I've validated these with IMS, and found that they all load, except `SURE_CHEMBL/LINKSET_EXACT_SURE_CHEMBL20160112.ttl`, e.g.

```turtle
<http://ops.rsc.org/OPS1931331> skos:exactMatch <http://rdf.ebi.ac.uk/resource/chembl/molecule/SCHEMBL170241> .
<http://ops.rsc.org/OPS1931518> skos:exactMatch <http://rdf.ebi.ac.uk/resource/chembl/molecule/SCHEMBL167446> .
<http://ops.rsc.org/OPS1931950> skos:exactMatch <http://rdf.ebi.ac.uk/resource/chembl/molecule/SCHEMBL175877> .
```

This is the most important one as it links the `SCHEMBL` identifiers to the `OPS` identifiers - thus linking SureChembl to the rest.

The `DataSources.txt` in IMS needs to be modified to support the new URI pattern for SureChembl. Nothing is currently resolved from those URIs - @agaulton - are they going to remain as those URIs?

**Comment by agaulton** [comment]
The URIs used in the SureChEMBL RDF should be like: <http://rdf.ebi.ac.uk/resource/surechembl/molecule/SCHEMBL175877> rather than resource/chembl/molecule.  So looks like there is an issue in the linkset? We have no plans to change our URIs in the foreseeable future though.
Anna

**Comment by nicklynch** [comment]
@batchelorc Any thoughts on linkset comments here?

**Comment by stain** [comment]
Thanks, @agaulton - I'll add http://rdf.ebi.ac.uk/resource/surechembl/molecule/SCHEMBL175877 (still 404) together with https://www.surechembl.org/chemical/SCHEMBL175877

BTW, if I do 

```
curl -v -H "Accept: application/rdf+xml" https://www.surechembl.org/chemical/SCHEMBL175877
```
.. I get a 500 Internal Server error :-(  (I guess it should redirect to the rdf.ebi.ac.uk resource, even if it's still 404 Not Found)

**Comment by batchelorc** [comment]
Hello chaps,

Well spotted! We need to fix this, so I’ll put a JIRA item in before we generate the RDF.

All the best,
Colin.

**Comment by agaulton** [comment]
Yep, we don’t have the SureChembl RDF in the EBI RDF platform yet, so the URIs won’t resolve at present.

Are we using the SureChEMBL interface URLs somewhere in the RDF/Open PHACTS (e.g. https://www.surechembl.org/chemical/SCHEMBL175877)? I.e., is there a reason we should expect this to resolve to RDF?

Cheers
Anna

**Comment by danidi** [comment]
From a user perspective, adding the interface URL to the IMS helps to retrieve data from the API. E.g. you could search on the SureChEMBL page and the use this URL directly in the API.

**Comment by agaulton** [comment]
OK that makes sense, I just wasn’t sure if it came from our side...
A

**Comment by madgpap** [comment]
:+1:

**Comment by stain** [comment]
Agree on @danidi - also the www URI will be coming back from IMS as part of the response when looking up any of the URIs that lead to it through the RSC linkset; which could be useful for users to click on in a browser. 

The redirect from www to rdf is just about being good web citizen (once) you have the RDF live. I expect it fails with a 500 (rather than 406 Not Acceptable) is because you have partially got the redirection functionality there?
